computation-slurm/src/main/java/com/powsybl/computation/slurm/SlurmComputationManager.java

```diff
@@ -330,14 +353,24 @@ private void scancel(Long jobId) {
                 Thread.currentThread().interrupt();
             }
 
-            if (closeStarted) {
-                return;
-            }
-
             SlurmExecutionReport report = generateReport(jobIdCommandMap, remoteWorkingDir);
-
-            R result = handler.after(remoteWorkingDir, report);
-            f.complete(result);
+            if (f.isCancelled()) {
+                if (f.isCancelledBySlurm()) {
+                    // some exceptions in platform
+                    handler.after(remoteWorkingDir, report);
```

Comment:
I think we should not call this even when cancelled by Slurm : the content will still not be the one expected from a valid computation, and we have no way to know that the computation has been interrupted.
In future evolutions, I think we should restore the call with the additional information that the computation has been interrupted.

Reply:
As discussed just now : it looks OK to leave this call here. The `ExecutionReport` will report an invalid exit code, so the implementation knows that something wrong happened.

However, we must be more clear in powsybl-core javadoc about what implementations should take care of (in particular, assuming the command may be interrupted).